src/demo.py

```diff
@@ -65,7 +69,7 @@ async def _main():
     password = args.password or getpass("Password: ")
 
     async with aiohttp.ClientSession() as session:
-        opower = Opower(session, args.utility, username, password)
+        opower = Opower(session, args.utility, username, password, args.mfa_secret)

```

Comment:
Can you call requires_mfa to demonstrate how it will be used in Home Assistant? Maybe call it at line 70?

Reply:
I removed requires_mfa because it was a somewhat insufficient/demonstration only. It doesn't tell us anything useful/we can't rely on it since even ConEd looks like it supports MFA-less accounts (for very old accounts, I assume). It was really only useful to paper over the difference in parameter count for the login method.